.github/workflows/docker-build-image.yml

```diff
@@ -0,0 +1,46 @@
+name: HDX/CKAN create image
+
+on:
+  push:
+    branches:
+      - dev
+      - prod
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        id: checkout
+        uses: actions/checkout@v2
+
+      - name: Tag Info
+        id: tag_info
+        run: |
+          export IMAGE_TAG="${GITHUB_REF#refs/*/}"
+          export IMAGE_TAG=${IMAGE_TAG//[^[:alnum:].-]/-}
+          echo ::set-output name=IMAGE_TAG::${IMAGE_TAG}
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.ECR_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.ECR_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Login to Public ECR
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```

Comment:
Can we use a specific tag/versino of the _docker/login-action_ command ? If not, could you please add a comment with why we need this specific version ?

Reply:
yep sure, pushed to use the generic v1